build/configs/artik055s/audio/defconfig

```diff
@@ -846,7 +846,8 @@ CONFIG_AUDIO_BUFSIZE=2048
 #
 # Exclude Specific Audio Features
 #
-# CONFIG_AUDIO_EXCLUDE_VOLUME is not set
+# CONFIG_AUDIO_EXCLUDE_MIC_GAIN is not set
+# CONFIG_AUDIO_EXCLUDE_SPEAKER_VOLUME is not set
```

Comment:
CONFIG_AUDIO_EXCLUDE_VOLUME & CONFIG_AUDIO_EXCLUDE_GAIN is correct
@btheosam  let us hear your opinion

Reply:
I sent the review request to @btheosam.

Reply:
Now, there are CONFIG_AUDIO_EXCLUDE_VOLUME and CONFIG_AUDIO_EXCLUDE_GAIN.
Is it okay?